assets/layers/cycleways_and_roads/cycleways_and_roads.json

```diff
@@ -21,6 +21,12 @@
         "highway=unclassified",
         "highway=primary",
         "highway=secondary",
+        "highway=tertiary_link",
+        "highway=primary_link",
+        "highway=secondary_link",
+        "highway=service",
+        "highway=footway",
+        "highway=pedestrian",
```

Comment:
IMHO living_street is missing too

Reply:
Added.

Some other suggestions (mostly by GitHub Copilot)

```json
"highway=steps",
"highway=path",
"highway=track",
"highway=bridleway",
"highway=bus_guideway",
"highway=escape",
"highway=motorway",
"highway=motorway_junction",
"highway=trunk",
"highway=trunk_link",
"highway=road",
"highway=busway"
```

path and track would make sense, considering that some regions have a lot of tracks which are primarily used for cycling and are tagged with `tracktype=grade1` and `surface=asphalt`. Adding path would mess up some stuff like the title rendering.

Reply:
Proposed by GH copilot? Nice...

I also wrote a script to read the id-presets, that might also work well for roads.

We'll discuss it all IRL monday.